resources/assets/js/app/views/votes/CreateVote.js

```diff
@@ -111,6 +111,9 @@ module.exports = Marionette.LayoutView.extend({
         var users = [];
         var tags = [];
 
+        if(view.model.get('finished_at') == '0000-00-00 00:00:00' || view.model.get('finished_at') == '')
```

Comment:
Probably better to create helper function in date helpers and check is date empty there